masonite/response.py

```diff
@@ -87,6 +89,10 @@ def view(self, view, status=200):
 
         if isinstance(view, dict) or isinstance(view, list):
             return self.json(view)
+        elif isinstance(view, Collection):
+            view = view.serialize()
```

Comment:
looking at this again, this line should also be JSON `view = self.json(view.serialize())`